### ISSUE TITLE
No files are served with InstallShield deployment of VSTO project

### ISSUE THREAD

**Issue by GilPalma** [jgauffin/Griffin.WebServer#28]
Let me start by thanking you for your webserver. It has been helpful with a project I'm working on but it seems I've hit a wall. 

For a while I've been using clickonce deployments to get my VSTO project running on other computers but that method is no longer viable. Therefore, I started using InstallShield to deploy my project (I haven't been able to successfully use the normal Visual Studio setup projects) but since then I've been met with some problems. While so far these problems have been fixable, I cannot find the solution to the most recent issue.

Once my VSTO project has been installed on a non-dev machine with the InstallShield method, the web files that the Webserver should be serving appear to be empty, according to the data shown in Telerik Fiddler. No web files reach their destination despite them being correctly installed in the file system (they appear in the right directory and match the development versions). Furthermore, no error messages pop up at all. The C#/.NET components load perfectly but the web files never seem to load.

Due to the lack of documentation, I haven't been able to understand what the problem is and so, while I try to find a solution myself, I decided to ask for help here in case there's something simple/direct I'm missing in the Webserver settings on my code.

Thank you for your time.

**Comment by jgauffin** [comment]
Can you check if the files are empty? Are the path that the webserver is using pointing to the same directory as for the files being installed? Do the webserver have access to that folder?

**Comment by GilPalma** [comment]
The files are not empty as I'm able to find them in their installation folder, as files in the windows file system, and by navigating to their localhost path through a non-office-control browser, such as Chrome or standalone IE, and I can see they have content, in both cases, which seems to mean that: 1) the files are being correctly installed and have the right content; 2) Griffin Webserver seems to be delivering the files since I can inspect and download them in a normal browser.

This problem is that nothing shows up when the page loads and this only happens in non dev computers. This doesn't happen in my development environment and everything runs smoothly. This also didn't happen when I used clickonce deployment, which makes me think this is a very specific problem between Griffin Webserver and InstallShield, even though I have no solid proof to support this.

**Comment by GilPalma** [comment]
I decided to try and create a brand new setup project, following all the steps a few more times to make sure the project was built correctly.
Not sure what I did differently or if the problem was fixed with a software update but the problem no longer occurs. I apologize for wasting your time and not providing with a clear answer/solution but I really can't tell what changed.
Thank you for your attention. Feel free to close this thread.
